requests-for-comments/2024-08-14-youtuber.md

```diff
@@ -0,0 +1,53 @@
+
+- Create video social content + changelog like so many people have wanted to.
+- Experiment with short form content.
+- Collaborate with influencers (we have done this with newsletters and it works well). This would also help with my influencer plan.
+- Read out 20,000 names (just kidding… unless…).
+
+This does require the **right person**, a proper YouTuber, probably someone who already is one now. I think this is a really cool role for them and PostHog is the right company to attract them. We could talk to the YouTubers we are working with now. 
+
+**What does great look like?** YouTube grows as a source of signups. YouTuber is publishing multiple videos per week. Videos are consistently hitting 10k+ views, some viral hits. Developing video as expertise at PostHog.
+
+**Why not another (writing) content person?** We need to fill for James. There is always more newsletters, docs, tutorials, and blogs to do, but I don’t feel it is enough to add another content person (beyond filling in for James).
+
+## We shouldn’t wait for the community manager
+
+Review comments:
+
+> We should wait for the community manager
+
+I do see where you are coming from on a community manager front, this will likely overlap with what their work is, but I disagree on it being the reason we should not hire someone here reasons.
+
+1. The job listing for community manager went out June 7th-ish. This shows it takes a long time to hire for a role. It will take them more time to decide to join, onboard, and get used to the PostHog culture. It is a big unknown, both in terms of what they want to do and when it will happen. 
+
+2. Unless I am misunderstanding, nowhere in the community manager role does it say they will be creating the content or deciding what content we will be creating. I think content creation sits squarely in the marketing camp even though we will of course be collaborating and working with the community manager.
+
+3. I do think the YouTuber will have a better idea of what content will work on YouTube. We already have some sense of this, but don’t have the bandwidth to execute. For example, more content like: [demo](https://www.youtube.com/watch?v=2jQco8hEvTI), [Clickhouse vs Postgres](https://www.youtube.com/watch?v=iLXXoDaFoxs), [event tracking guide](https://www.youtube.com/watch?v=LIJ_TuyMq74), [modern data stack sucks](https://www.youtube.com/watch?v=2N2cvCmv4us). The community manager will also be scattered with other initiatives and not dedicate themselves to YouTube like I am suggesting. 
+
+4. I still see influencer work sitting entirely with the community manager. It makes it EASIER for them to do their job by having a YouTuber to collaborate with. It brings us to the attention of more YouTubers. 
+
+I’m perfectly fine with this being a “no, we shouldn’t hire a YouTuber” but I don’t think “we should let the community manager decide” is the right answer. Other ones that I think could be better reasons:
```

Comment:
Yeah I agree that the two roles don't need to be coupled - we have enough informed and strong opinions that a relatively new person doesn't need to come in and tell us who to hire (and they will sit on different teams).

Reply:
This all makes sense to me. I'd suggest:

1. We should at least wait until the upcoming SuperDays are completed. If we end up hiring one of them, they can provide some input before we start advertising.

2. If none of them make it, we should get an ad up for this because it could take a while to find someone suitable, so delaying would be bad.

3. Before we go live with an ad, I think we should spend a bit more time forming an opinion on what we want. We can also look at how other companies like us do YouTube, such as Supabase and Arc. I'd go so far as coming up with a long list of potential video ideas, just to get us thinking about this as a channel before we 100% commit.